perun-openapi/openapi.yml

```diff
@@ -3639,6 +3639,21 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/attributesManager/getAttributes/user-facility:
```

Comment:
This method is already in the specification aa `/json/attributesManager/getAttributes/f-u`